src/history/history-detail-overview.component.tsx

```diff
@@ -28,8 +31,59 @@ const resources: string[] = [
 
 const HistoryDetailOverview: React.FC<HistoryDetailOverviewProps> = () => {
   const { t } = useTranslation();
+  const abortController = new AbortController();
+  const [isSubmitting, setIsSubmitting] = useState(false);
   const { patientUuid: uuid } = usePatient();
-  const { summaries, error, isLoading } = useIpsResource(uuid);
+  const { history, error, isLoading } = useIpsResource(uuid);
+
+  const [ips, setIps] = useState({
+    history: null,
+    isLoading: true,
+    error: "",
+  })
+  
+  useEffect(() => {
+    if (history != null) {
+      setIps((ps) => ({...ps, history: history, isLoading: isLoading, error: error?.message}));
+    }
+  },[error, history, isLoading])
+
+  console.error(ips);
+
+  const handleSubmit = () => {
+    setIsSubmitting(true);
+    createIpsResource(
+      uuid, 
+      abortController,
+    ).then((response) => {
+      if (response.status === 200) {
+        setIps((ps) => ({...ps, history: history, isLoading: isLoading, error: error?.message}));
+        setIsSubmitting(false);
+        showSnackbar({
+          isLowContrast: true,
+          kind: 'success',
+          title: t('ipsCreated', 'IPS bundle generated successfully!'),
```

Comment:
Can we change this to have the title as 'IPS' and then the subtitle as  'IPS bundle downloaded successfully'